src/api.v2/controllers/experimentController.js

```diff
@@ -8,33 +8,43 @@ const { OK, NotFoundError } = require('../../utils/responses');
 const sqlClient = require('../../sql/sqlClient');
 
 const getExperimentBackendStatus = require('../helpers/backendStatus/getExperimentBackendStatus');
+const Sample = require('../model/Sample');
+const MetadataTrack = require('../model/MetadataTrack');
 
 const logger = getLogger('[ExperimentController] - ');
 
 const getAllExperiments = async (req, res) => {
   const { user: { sub: userId } } = req;
+  console.log(`Getting all experiments for user: ${userId}`);
 
   const data = await new Experiment().getAllExperiments(userId);
 
+  console.log(`Finished getting all experiments for user: ${userId}, length: ${data.length}`);
+  res.json(data);
+};
+
+const getAllExampleExperiments = async (req, res) => {
```

Comment:
in the end we agreed we won't have distinction between public & example right? because then this function should probably be called `getPublicExperiments`. (btw I think the `All` is rarely needed, you just want the experiments matching something, makes the name unnecessarily longer IMO)

Reply:
Right now this endpoint gets example experiments, and it is being used in this way in the UI too, if/when we start using public experiments is when I think renaming this would be better I think